src/test/java/org/casbin/jcasbin/main/RbacAPIWithDomainsUnitTest.java

```diff
@@ -98,4 +98,28 @@ public void testImplicitPermissionAPIInDomain() {
 
         testGetImplicitPermissionsInDomain(e, "alice", "domain1", asList(asList("admin", "domain1", "data1", "read"), asList("admin", "domain1", "data1", "write")));
     }
+    @Test
```

Comment:
Test cases should be added to the src/test/java/org/casbin/jcasbin/main/ModelUnitTest.java.